src/qt/forms/receivecoinsdialog.ui

```diff
@@ -108,7 +108,7 @@
              </size>
             </property>
             <property name="text">
-             <string>&amp;Request payment</string>
+             <string>&amp;Receive payment</string>
```

Comment:
This doesn't seem literally correct. Receiving only happens when the sender sends. :/

Reply:
Should this text even change than. As far as I understand it's function it makes sense.

Reply:
maybe call it something like "Create new receiving address" instead?

Reply:
Will do.